src/ui/public/courier/fetch/request/segmented.js

```diff
@@ -100,7 +107,7 @@ define(function (require) {
     };
 
     SegmentedReq.prototype.isIncomplete = function () {
-      return this._queue.length > 0;
+      return !this._queueCreated || this._queue.length > 0;
```

Comment:
That wouldn't be the same conditional logic though: this is only negating the first half of the condition.